add NVIDIA TXAA (#576)
SteamDatabase/FileDetectionRuleSets: descriptions/SDK.NVIDIA_TXAA.md

> [**TXAA**](https://www.nvidia.com/en-gb/geforce/technologies/txaa/) anti-aliasing creates a smoother, clearer image than any other anti-aliasing solution by combining high-quality MSAA multisample anti-aliasing, post processes, and NVIDIA-designed temporal filters.

Reviewer: ```suggestion
[**TXAA (Temporal anti-aliasing)**](https://www.nvidia.com/en-gb/geforce/technologies/txaa/) is an anti-aliasing that combines high-quality MSAA multisample anti-aliasing, post processes, and NVIDIA-designed temporal filters.
```
making it more neutral could be nice tho, i'm not a fan of that suggestion
